client/src/components/layout/DestinationsIndex.js

```diff
@@ -34,10 +35,13 @@ const DestinationsIndex = (props) => {
     
 
     return (
-        <>
-            <h1>Destinations!</h1>
-            {destinationTiles}
-        </>
+        <div className="grid-container">
+            <div className="grid-x grid-margin-x">
```

Comment:
don't need 2 enclosing divs - `grid-container` doesn't do much, and is best for high up top level components